src/components/Brand.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <a class="d-flex flex-row" href="https://jeodns.com/">
+    <p class="my-auto mx-2">JeoDNS</p>
+    <img src="@/assets/icons/jeo.svg"/>
+  </a>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  setup() {},
+});
+</script>
+
+<style lang="scss" scoped>
+a {
+  text-decoration: none;
+}
+
+p {
+  font-family: "Poppins";
```

Comment:
What is this font? it's installed on all of OSes,like Arial?
It's better to change default font of your app to Iransans.

Reply:
Used IRANSans as default font